File: Quiz.Android/lib/models/menu.dart
class Menu {
  final int menuId;
  final String name;
  final String image;
  final bool isActive;

  const Menu({
    required this.menuId,
    required this.name,
    required this.image,
    required this.isActive,
  });

  factory Menu.fromJson(Map<String, dynamic> json) {
    return Menu(
      menuId: json['menuId'],
      name: json['name'],
      image: json['image'],
      isActive: json['isActive'],
    );
  }
}

File: Quiz.Entity/Models/Menu.cs
using System.ComponentModel.DataAnnotations;

namespace Quiz.Entity.Models
{
    public class Menu
    {
        [Key]
        public int MenuId { get; set; }
        public int ParentId { get; set; }
        public string Name { get; set; }
        public string Image { get; set; }
        public bool IsActive { get; set; }
        public int SerialNo { get; set; }
    }
}
